queries.sql

```diff
@@ -54,4 +48,40 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts)
 FROM animals
 WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+SELECT name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
+
+SELECT species.name, COUNT(animals.id) as count
+FROM species
+LEFT JOIN animals ON species.id = animals.species_id
+GROUP BY species.id;
```

Comment:
The result's screenshot for this query is missing. Kindly include it in your PR before submitting it for the re-review.

Reply:
Thanks for review, I will be working to fix requested changes 👍